doc/examples/applications/plot_human_mitosis.py

```diff
@@ -0,0 +1,176 @@
+"""
+================================
+Segment human cells (in mitosis)
+================================
+
+In this example, we analyze a microscopy image of human cells. We use data
+provided by Jason Moffat [1]_ through
+[CellProfiler](https://cellprofiler.org/examples/#human-cells).
+
+.. [1] Moffat J, Grueneberg DA, Yang X, Kim SY, Kloepfer AM, Hinkle G, Piqani
+       B, Eisenhaure TM, Luo B, Grenier JK, Carpenter AE, Foo SY, Stewart SA,
+       Stockwell BR, Hacohen N, Hahn WC, Lander ES, Sabatini DM, Root DE
+       (2006) "A lentiviral RNAi library for human and mouse genes applied to
+       an arrayed viral high-content screen" Cell, 124(6):1283-98.
+       PMID: 16564017
+       :DOI:`10.1016/j.cell.2006.01.040`
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import ndimage as ndi
+
+from skimage import feature, filters, io, morphology, util
+
+
+image = io.imread('https://github.com/CellProfiler/examples/blob/master/ExampleHuman/images/AS_09125_050116030001_D03f00d0.tif?raw=true')
+
+#####################################################################
+# This image is a TIFF file. If you run into issues loading it, please
```

Comment:
This is a bit sad :-(. @mkcor added this comment because she ran in the same issue as in #4326. If we store our own version of this file should we convert it to png (hiding the tiff problem under the rug is not great either...)

Reply:
This should no longer be an issue if you `pip install -U tifffile`, because @cgohlke provided wheels for imagecodecs. So I would argue that we should remove the comment, and perhaps increase the tifffile version required for docs.

Reply:
on the other hand this comment is not for our build but for users who may have their own set of installed packages. Only a small fraction of users will think of consulting the doc dependencies for minimal versions when looking at a doc example.

Reply:
If an extra package is missing, then a proper message must be displayed. If it is not the case, it is a bug. If it is the case, I don't thing this comment is useful anymore.

Reply:
I'd probably get rid of this, given that it has been fixed.  Some users may run into it initially, but that number should drop down very quickly.

Reply:
Ok, done c5eb4c17710584b30a094ba772fd7310b1da5a8a. I wish I could check the behaviour for real but I haven't managed: For the past half-hour I've been trying to test this in a fresh environment, but the install (of scikit-image) kept failing. I've used pip, conda, and mamba -- only the latter pointed to a timeout issue while installing, while pip and conda would 'install' apparently 'successfully' but then I would find out the install was broken when loading the package.